src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Buttons
+description: >
+  A button is an element on a page or view that tells a user what they can do and triggers an action.
+---
+
+## Anatomy
+
+![Usage of button types](figma:patterns-buttons-anatomy-button-type.png)
+
+1. Primary buttons are for the most important action in a given view. Use one primary button per page or view.
+2. Default buttons are for secondary actions
+3. Basic buttons are for repetitive or deemphasized actions in a given view
+
+## Formatting
+
+### Size
+
+Button size plays a part in content and user interaction.
+Clear labels and visual signifiers such as shape, color, and size
+help communicate the types of actions available and their priority in the process.
+
+- The default button size is medium
+- Use small buttons on screens that have greater information density
+
+<BestPractice>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.buttonsSizeDo.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Use same size buttons and inputs in the same flow.</li>
+    </ul>
+  </Do>
+  <Dont
+    imageIsFreeForm
+    imageSource={props.data.buttonsSizeDont.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Avoid mixing and matching button and input sizes.</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### Alignment
+
+When a basic or default button is placed in conjunction with a primary button,
+the primary button should always be placed at the end in left-to-right layouts.
```

Comment:
```suggestion
the primary button should always be placed at the end.
```

Technically "the end" is always the end – whether in LTR or RTL :wink:.

Reply:
Agreed @jzempel. I originally had it as your recommendation here but Jen added those extra words when she reviewed the Google Doc. Will keep it as I originally had it.